How-we-use-pull-requests.md

```diff
@@ -0,0 +1,68 @@
+## Table of contents
+
+* [PR lifecycle](#pr-lifecycle)
+* [Draft pull requests](#draft-pull-requests)
+* [Labeling issues and pull requests](#labeling-issues-and-pull-requests)
+* [Juggling multiple pull requests](#juggling-multiple-pull-requests)
+
+## PR lifecycle
+
+![Flowchart of PR lifecycle](images/prLifecycle.png)
+
+<details>
+<summary>Flowchart source code</summary>
+
+The flowchart was generated from the below source code by https://flowchart.fun.
+
+```text
+~~~
+layout:
+  name: dagre
+  rankDir: LR
+~~~
+Draft PR
+  Ready for review: PR awaiting review
+    Request changes: Changes requested
+      Addresses comments: (PR awaiting review)
+    All reviewers approve: PR labeled LGTM
+      Merge: PR closed
+```
+</details>
+
+1. (optional) Draft PR: Authors can open draft PRs to get early feedback or debug CI tests. Reviews at this stage are at the discretion of the author.
+2. PR ready for review: Once authors mark PRs as ready for review (or if they skip the draft PR stage entirely), reviews will automatically be requested from all code owners. Oppiabot will assign one developer (typically the developer from the changelog label) to take a first pass. Oppiabot will assign the other code owners to review once the first pass is complete.
+3. Changes requested: Reviewers will either approve the PR or request changes. Almost all PRs will have at least some changes requested. PR authors address each comment, either by making the requested change or explaining why they think it is unnecessary. Once all comments are addressed, reviewers will re-review.
+4. LGTM: A PR gets labeled LGTM ("looks good to me") when it has sufficient reviews. This means that a code owner from each modified file has approved. When multiple developers share code ownership, only one has to approve.
+5. PR merged: PRs can be merged once CI checks pass and the LGTM label has been applied. Only members of the Oppia organization can merge PRs.
+
+## Draft pull requests
+
+While making a contribution, you may discover that your change is not complete and needs some more work. You can make a draft pull request to make it easy for other developers to give you feedback; however, draft pull requests consume resources like time on our CI runners, which slows down CI runs for other developers. Hence, you should prefix the commit messages with `[skip ci]` or `[ci skip]` to disable CI runs when those commits are going to a draft PR. If you want those tests to run to help you debug a problem, you should open a PR against your fork's `develop` branch instead of the `oppia/oppia` develop. This will still delay other developers' Circle CI runs, but not their GitHub Actions runs.
```

Comment:
Can you please check with the OppiaBot team on what are the bot's expectations from draft PRs? (I'm unable recall all the expectations!)

Reply:
Based on the recent draft PRs I've opened, I'm pretty sure that Oppiabot just expects to see `[skip ci]` or `[ci skip]` in the commit message.

@vojtechjelinek is this correct?

Reply:
@vojtechjelinek has confirmed offline that Oppiabot just expects to see `[skip ci]` or `[ci skip]`